en-US/snippets/2018/apr2018.lang

```diff
@@ -0,0 +1,61 @@
+## NOTE: Reference bug 1446192; Deadline: 9 April, 2018
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: cross platform promotions on a variety of topics
+## NOTE: Try to limit character length to 180, preferably under 160.
```

Comment:
Try to limit is never going to work IMO. You should put a MAX_LENGTH on each one, to make they don't go over 180 at least.

Reply:
let me fix this